detection_rules/packaging.py

```diff
@@ -270,8 +270,7 @@ def generate_summary_and_changelog(self, changed_rule_ids, new_rule_ids, removed
             if index_list:
                 indexes.update(index_list)
 
-        letters = ascii_uppercase + ascii_lowercase
-        index_map = {index: letters[i] for i, index in enumerate(sorted(indexes))}
+        index_map = {index: str(i) for i, index in enumerate(sorted(indexes))}
```

Comment:
Note: This was a bug. `letters` originally was all lower and uppercase letters used for creating an index map, however there is a maximum of 54 here. Thus as we add more indexes, this would threshold would have inevitably been reached and was in this PR.

Rather than use letters, we can just map to integers, making the threshold infinite.

I've ran the `python -m detection_rules dev build-release` command to generate a prebuilt rules package, where this code is touched, and the outcome seems fine. 
<img width="1883" alt="Screenshot 2024-06-26 at 11 57 24 AM" src="https://github.com/elastic/detection-rules/assets/99630311/11776cda-40c1-494d-8030-f90bb2fb91b2">
<img width="710" alt="Screenshot 2024-06-26 at 11 57 33 AM" src="https://github.com/elastic/detection-rules/assets/99630311/5c598df8-54f9-4315-bb27-6613c7d89a3d">
<img width="625" alt="Screenshot 2024-06-26 at 11 57 55 AM" src="https://github.com/elastic/detection-rules/assets/99630311/5daa7dfa-9993-4ca7-9f73-9146e5d672f0">

@shashank-elastic @eric-forte-elastic can we confirm?

Reply:
Yes this limit would be hit in current limitation, we had a proposed fix [here](https://github.com/elastic/detection-rules/pull/3826), but am more inclined to using this method of fix as well!